bot/src/index.js

```diff
@@ -1,57 +1,48 @@
-const Feed = require("feed-to-json-promise");
 const { Client } = require("discord.js");
 
-const client = new Client();
-const feed = new Feed();
+const commands = require("./commands/index");
+const { getFeed } = require("./utilities/getFeed");
 
-const serverID = "458034851296313375";
+const client = new Client();
+const feedList = ["https://xkcd.com/rss.xml"];
+const serverId = "458034851296313375";
 
 client.on("ready", async () => {
   console.log(`Logged in as ${client.user.tag}!`);
-  const rssFeed = "https://xkcd.com/rss.xml";
-  const channelId = await addFeedChannel(rssFeed, serverID);
-  try {
-    const items = await getFeedItems(rssFeed);
-    sendFeedItems(items, channelId);
-  } catch (error) {
-    console.error(error);
+  for (const rssFeed of feedList) {
+    try {
+      await commands.add([rssFeed], serverId, client);
+    } catch (error) {
+      console.error(error);
+    }
   }
 });
 
 client.on("message", msg => {
   if (msg.content === "ping") {
     msg.reply("pong");
   }
+  if (!msg.mentions.users.size) {
+    return;
+  } else if (msg.mentions.users.first().username === "RnD") {
+    const args = msg.content.split(/\s+/);
+    args.shift();
+    runCommand(args, msg);
+  }
 });
 
-client.login(process.env.BOT_TOKEN);
+try {
+  client.login(process.env.BOT_TOKEN);
+} catch (error) {
+  console.log(error);
+}
 
-const addFeedChannel = async (url, guildId) => {
-  const rssFeed = await feed.load(url);
-  const guild = client.guilds.find(guild => guild.id === guildId);
-  const channelName = rssFeed.title.toLowerCase().replace(/[^a-z0-9]/g, "");
-  const channel = guild.channels
-    .filter(ch => ch.type === "text")
-    .find(ch => ch.name === channelName);
-  if (!channel) {
-    const newChannel = await guild.createChannel(channelName, "text");
-    return newChannel.id;
-  }
-  return channel.id;
-};
-
-const getFeedItems = async url => {
-  const rssFeed = await feed.load(url);
-  return rssFeed.items;
-};
+const runCommand = async (args, msg) => {
+  const [cmd, ...rest] = args;
+  const result =
+    !commands[cmd] || cmd === "help" || cmd === "-h"
```

Comment:
Instead of special-casing `cmd === "-h"`, you can alias the handler in `commands`:
```js
// commands/index.js
{
  help,
  '-h': help,
  ...
}
```

Then use `||` to make `help` the default:
```js
const command = commands[cmd] || commands.help;
const result = await command(rest, serverId, client);
```